scripts/deploy/site.yml

```diff
@@ -6,10 +6,16 @@
     raito_working_dir: /home/raito/raito-bridge-node
     raito_user: raito
     raito_service_name: raito-bridge-node
+    # Proof builder configuration
+    proof_builder_user: maciej
```

Comment:
Why not raito user?

Reply:
Because it already installed there. I fell at the current stage of the project there is no need to separate the deployment under dedicated user. We want/need to be able to do manual experiments with proving process, which it possible only on the server.